packages/loader/driver-utils/src/parallelRequests.ts

```diff
@@ -426,7 +426,9 @@ async function getSingleOpBatch(
 
 	while (signal?.aborted !== true) {
 		retry++;
-		let delay = Math.min(MaxFetchDelayInMs, MissingFetchDelayInMs * Math.pow(2, retry));
+		let defaultDelay = MissingFetchDelayInMs;
```

Comment:
I think this needs to be outside of while loop - it resets the value on every iteration. If that's the intention, then we do not need this variable here at all (maybe down the line only)

Reply:
Fixed.